pkg/virt-api/definitions/definitions.go

```diff
@@ -582,6 +582,13 @@ func MoveCursorParam(ws *restful.WebService) *restful.Parameter {
 	return ws.QueryParameter(MoveCursorParamName, "Move the cursor on the VNC display to wake up the screen").DataType("boolean").DefaultValue("false")
 }
 
+func ForceParam(ws *restful.WebService) *restful.Parameter {
+	return ws.
+		QueryParameter("force", "Enforce behavior that otherwise would not be allowed.").
```

Comment:
```suggestion
		QueryParameter(ForceParamName, "Enforce behavior that otherwise would not be allowed.").
```

Add const for `"force"`?